### PR TITLE
Attack() method (missing tests)

### PR DESCRIPTION
**Pull request by Kerry-G**

### CODE REVIEW

File: src/Map/Map.cpp [hunk 1]
```diff
@@ -37,6 +39,14 @@ namespace Board {
         *nb_armies = num;
     }
 
+    void Country::decrement_army() {
+        *nb_armies = *nb_armies - 1;
+    }
+
+    void Country::increment_army() {
+        *nb_armies = *nb_armies + 1;
+    }
```
Comment (Nwitz) [inline src/Map/Map.cpp:48]: Maybe a method to add and remove a given number of armies as well?

File: src/Map/Map.cpp [hunk 2]
```diff
@@ -46,6 +56,34 @@ namespace Board {
...
+
+    vector<Country *> *Country::get_neighbors() const {
+        return neighboring_countries;
+    }
+
+    string Country::to_string_with_neighbors() {
+        ostringstream os;
+        os << "name: " << *name << ", owner: " << owner << ", number of armies: " << *nb_armies << "\n";
+        for (Country *neighbor: *neighboring_countries) {
```
Comment (Nwitz) [inline src/Map/Map.cpp:80]: This method will break, but I have fixed it in my branch, can be left as is

File: src/Map/Map.cpp (hunk 1 above)
Comment (Kerry-G) [reply src/Map/Map.cpp:48]: Agreed, but for attack(), I only needed +1 and -1  😃

File: src/Map/Map.cpp (hunk 2 above)
Comment (Kerry-G) [reply src/Map/Map.cpp:80]: Perfect, we'll be careful on the merging

File: src/Map/Map.cpp (hunk 1 above)
Comment (Nwitz) [reply src/Map/Map.cpp:48]: Fair enough, will be added when needed

**Review by Nwitz — APPROVED** [review]

**Pull request merged**